modules/ui_extra_networks.py

```diff
@@ -188,12 +212,12 @@ def create_html(self, tabname, upload_button_id, button_id=None, return_callback
 </button>
 """ for subdir in subdirs])
 
-        for item in self.list_items():
-            metadata = item.get("metadata")
-            if metadata:
-                self.metadata[item["name"]] = metadata
+        # for item in list(self.list_items()):
+        #     metadata = item.get("metadata")
+        #     if metadata:
+        #         self.metadata[item["name"]] = metadata
 
-            items_html += self.create_html_for_item(item, tabname)

```

Comment:
Remove this line is fine, since you do not need to add the items any more. But you would be better to keep the for loop that updates the metadata.